easybuild/easyconfigs/t/Tensorflow/Tensorflow-1.2.1-foss-2016b-GPU-Python-3.5.2.eb

```diff
@@ -0,0 +1,64 @@
+easyblock = 'Bundle'
+
+name = 'Tensorflow'
```

Comment:
@schiotz We renamed this to `TensorFlow` (i.e. the official "spelling") in #5495, do you mind making that change here too?

Reply:
Done!